compiler/optimizer/OMROptimizer.cpp

```diff
@@ -2787,7 +2789,8 @@ void OMR::Optimizer::doStructureChecks()
 
 bool OMR::Optimizer::getLastRun(OMR::Optimizations opt)
    {
-   TR_ASSERT(_opts[opt], "Optimization manager for %d should be initialized first", opt);
+   if (!_opts[opt])
```

Comment:
ah, ok, that makes sense. I foolishly didn't think about opts asking about other opts :( .